README.md

```diff
@@ -45,6 +45,8 @@ feed:
 
 To note, you shouldn't have to do this unless you already have a feed you're using, and you can't or wish not to redirect existing subscribers.
 
+<span role="img" aria-label="Warning">⚠️</span> If your feed path does not end with `.xml`, you'll need to configure your web server to serve the proper content type. Alternatively, you can specify e.g. `feed/index.xml` to serve your feed from `/feed/`.
```

Comment:
I believe using `:warning:` here will result in similar markup.

Reply:
VoiceOver on macOS reads it slightly differently. However, I’ve noticed that GitHub converts it to their `<g-emoji>` custom element anyway, so I’ve changed it to `:warning:`. Thanks 👍